data/multiplayer/scenarios/2p_Isle_of_Mists.cfg

```
@@ -0,0 +1,298 @@
+        [/store_unit]
+        [if]
+            [variable]
+                name=leader.length
+                less_than=2
+            [/variable]
+            [then]
+                {VARIABLE lose_condition_string ( _ "Death of your leader")}
+                {VARIABLE notes ( _ "Since your team has only one leader, the enemy waves will not be at full strength.")}
+            [/then]
+            [else]
+                {VARIABLE lose_condition_string ( _ "Death of both of your team’s leaders")}
+                {VARIABLE notes ( _ "Since your team has two leaders, the enemy waves will be at full strength.")}
+                # each village will now support an upkeep of 2 instead of 1
+                # this makes the scenario somewhat passable now.
+                [modify_side]
+                    side=3,4
+                    village_support=2
+                [/modify_side]
+            [/else]
+        [/if]
+        {CLEAR_VARIABLE leader}
+        [music]
+            name=wanderer.ogg
+            ms_before=2000
+        [/music]
+        [music]
+            name=battle.ogg
+            ms_before=2000
+            append=yes
+        [/music]
+        [music]
+            name=breaking_the_chains.ogg
+            play_once=yes
+            immediate=yes
+        [/music]
+
+        [objectives]
+            [objective]
+                description= _ "Survive and defeat all enemy waves"
+                condition=win
+            [/objective]
+            [objective]
+                description=$lose_condition_string
+                condition=lose
+            [/objective]
+
+            [note]
+                description=$notes
+            [/note]
+        [/objectives]
+
+        {CLEAR_VARIABLE lose_condition_string}
+        {CLEAR_VARIABLE notes}
+
+        [label]
+            x,y = 16,9
+            text = _"Healing Keep"
+        [/label]
+
+        [unit]
+            side=5
+            type="Elvish Lady"
+            id="healer_elf"
+            x,y=16,9
+            canrecruit=no
+            moves=0
+            max_moves=0
+            hitpoints=28
+            max_hitpoints=28
+            generate_name=yes
+            [modifications]
+                {TRAIT_INTELLIGENT}
+                {TRAIT_DEXTROUS}
+            [/modifications]
+        [/unit]
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            caption= _"Isle of Mists — a random survival scenario"
+            speaker=narrator
+            image="portraits/monsters/fire-dragon.png"
+            message={ISLE_OF_MISTS_MSG_STR}
+        [/message]
+        [message]
+            side=3
+            message= _"Invaders approach on the horizon! Let us prepare defences!"
+        [/message]
+        [message]
+            side=4
+            message= _"The invaders are rumored to be a strange alliance of races, and our control over this isle might be threatened. We must be prepared for the surprises thrown at us."
```

```suggestion
            message= _"Phantom armies approach on the horizon. If we defeat them all, they may be released from this timeless prison."
```

If singleplayer, the player's leader should say this line.